src/Sarif.Converters/StaticDriverVerifierConverter.cs

```diff
@@ -66,24 +67,18 @@ private Result ProcessSdvDefectStream(Stream input)
             var result = new Result();
 
             result.Locations = new List<Location>();
-
-            result.CodeFlows = new List<CodeFlow>();        
-            result.CodeFlows.Add(new CodeFlow
-            {
-                Locations = new List<AnnotatedCodeLocation>()
-            });
-
-            var sb = new StringBuilder();
```

Comment:
Since you're no longer using `StringBuilder`, you probably don't need the `using` directive for `System.Text`.